package.json

```diff
@@ -26,16 +26,16 @@
     "@angular/platform-browser-dynamic": "^9.1.9",
     "@angular/router": "^9.1.9",
     "@angular/service-worker": "^9.1.9",
-    "@ngneat/until-destroy": "^7.1.6",
-    "@sentry/browser": "^5.16.0",
+    "@ngneat/until-destroy": "^7.3.0",
```

Comment:
Are those updates necessary for your PR? If not strictly required, let's reset package.json and package-lock.json to the master branch version and do updates in a separate, isolated PR. That makes it easier to track changes and avoids merge conflicts